CONTRIBUTING.md

```diff
@@ -27,4 +27,4 @@ The OSConfig Core team will respond to a PR that passes all checks in 3 business
 
 # Contact
 
-You may contact the OSConfig Core team at [osconfigcore@microsoft.com](mailto:osconfigcore@microsoft.com) to ask questions about OSConfig, to report bugs, to suggest new features, or inquire about any other OSConfig related topic.
```

Comment:
Please do not remove this

Reply:
@MariusNi 
The CONTRIBUTING.md is not deleted. If you look at the diff the same content is there.
The diff shown here is rendered 'funny' because there was no new line at the end of file.
The rule `end-of-file-fixer` worked and added new line, hence it altered the 
```
-You may contact the OSConfig Core team at [osconfigcore@microsoft.com](mailto:osconfigcore@microsoft.com) to ask questions about OSConfig, to report bugs, to suggest new features, or inquire about any other OSConfig related topic.
```
by adding empty new line at the end of file.
I would prefer to have that rule b/c on Linux if you use cat fileName.txt it will render, lastly the empty new line and then your normal prompt line will be shown.
See difference between
/usr/bin/printf 'This is a test\n'
vs 
/usr/bin/printf 'This is a test'
(printf should do not add new line)

So we can consider:

Do we want this rule? 
If we do wan't it maybe the files to which it's applied should be refined - to be more line in mixed-line-ending case

Reply:
Hi @MariusNi the CONTRIBUTING.md is not removed as you can see https://github.com/robertwoj-microsoft/azure-osconfig/blob/kkanas/add-formating-check/CONTRIBUTING.md